.github/workflows/scripts/test-with-nu-quickstart.sh

```diff
@@ -3,22 +3,24 @@
 cd "$(dirname "$0")"
 
 cd ../../../
-EXAMPLES_LIBABRY_BUILD_TEMP_VERSION=$(uuidgen | tr -d '-' | tr 'A-Z' 'a-z' | cut -c1-10)
-echo "1. Building Scenario Examples Library image. Version: ${EXAMPLES_LIBABRY_BUILD_TEMP_VERSION}..."
-docker buildx build --tag touk/nussknacker-example-scenarios-library:"$EXAMPLES_LIBABRY_BUILD_TEMP_VERSION" .
+EXAMPLES_LIBRARY_BUILD_TEMP_VERSION=$(uuidgen | tr -d '-' | tr 'A-Z' 'a-z' | cut -c1-10)
+echo "1. Building Scenario Examples Library image. Version: ${EXAMPLES_LIBRARY_BUILD_TEMP_VERSION}..."
+docker buildx build --tag touk/nussknacker-example-scenarios-library:"$EXAMPLES_LIBRARY_BUILD_TEMP_VERSION" .
 
 echo "2. Checking out Nu Quickstart..."
 cd .github/workflows/scripts
 rm -rf nussknacker-quickstart
 git clone https://github.com/TouK/nussknacker-quickstart.git
 cd nussknacker-quickstart
-git checkout staging # TODO: change to main when Nu 1.17 is released
+git checkout main
```

Comment:
Why it is changed? What it give to us?

Reply:
Reverted (nussknacker-quickstart now uses "staging-latest" version of NU, with published image).